### PR TITLE
937 config calibration

### PR DESCRIPTION
**Pull request by JIMENOFONSECA**
It fixes 937.

### CODE REVIEW

**Comment by JIMENOFONSECA** [comment]
@BhargavaKrishnaSreepathi coudl you review this for me? I added a readme to the section cea/demand/calibration/bayesian calibration on how to test this. thanks!

File: cea/demand/calibration/bayesian_calibrator/calibration_gaussian_emulator.py [hunk 1]
```diff
@@ -33,6 +32,18 @@ def gaussian_emulator(locator, samples, cv_rmse, building_name):
...
+
+    # Local variables
+    building_name = config.single_calibration.building
+    building_load = config.single_calibration.load
+    with open(locator.get_calibration_problem(building_name, building_load),'r') as input_file:
+        problem = pickle.load(input_file)
+    samples_norm = problem["samples_norm"]
+    rmse = problem["cv_rmse"]
+    import pandas
+    pandas.DataFrame({'cvrmse':problem["cv_rmse"], 'rmse':problem["rmse"]}).to_csv(r'C:\Users\JimenoF\Desktop/test3.csv')
```
Comment (BhargavaKrishnaSreepathi) [inline cea/demand/calibration/bayesian_calibrator/calibration_gaussian_emulator.py]: Prevent hard coded csv file names. Link this to the inputlocator

File: cea/demand/calibration/bayesian_calibrator/calibration_main.py [hunk 2]
```diff
@@ -38,95 +37,79 @@
...
+            trace = pm.sample(20000, tune=1000, step=step)
+            # Saving
+            df_trace = pm.trace_to_dataframe(trace)
+
+            #CREATE GRAPHS AND SAVE TO DISC
+            df_trace.to_csv(locator.get_calibration_posteriors(building_name, building_load))
+            pm.traceplot(trace)
+
+            columns = ["var1", "var2", "var3", "var4", "var5", "var6"]
+            sns.pairplot(df_trace[columns])
```
Comment (BhargavaKrishnaSreepathi) [inline cea/demand/calibration/bayesian_calibrator/calibration_main.py]: sns is not being imported, so this line does not work

**Comment by BhargavaKrishnaSreepathi** [comment]
the PR runs through and generates plots. Make sure to include pymc3 and seaborn in the CEA installation as importing these modules is necessary for this PR

**Comment by daren-thomas** [comment]
@BhargavaKrishnaSreepathi pymnc3 should be there already, i'll add seaborn.

**Comment by daren-thomas** [comment]
@JIMENOFONSECA the calibration_sampling takes long - i'd like to be able to add it to the jenkins. any ideas on how to make it run in less time?

**Comment by daren-thomas** [comment]
I get this when running the gaussian emulator:

image

File: cea/demand/calibration/bayesian_calibrator/calibration_gaussian_emulator.py (hunk 1 above)
Comment (JIMENOFONSECA) [reply cea/demand/calibration/bayesian_calibrator/calibration_gaussian_emulator.py]: fixed

File: cea/demand/calibration/bayesian_calibrator/calibration_main.py (hunk 2 above)
Comment (JIMENOFONSECA) [reply cea/demand/calibration/bayesian_calibrator/calibration_main.py]: fixed

**Comment by JIMENOFONSECA** [comment]
@BhargavaKrishnaSreepathi @daren-thomas the changes have been addressed

**Comment by JIMENOFONSECA** [comment]
@daren-thomas to make the sampler run faster change the values config.single_calibration.samples = 10

**Comment by daren-thomas** [comment]
@JIMENOFONSECA works for me too. I added a small change that makes this a part of the dodo tests for the jenkins.

**Comment by daren-thomas** [comment]
merge?

**Pull request merged**